moped-editor/src/views/projects/newProjectView/MapDrawToolbarButton.js

```diff
@@ -1,42 +1,49 @@
 import React from "react";
-import { makeStyles, Tooltip } from "@material-ui/core";
-import theme from "../../../theme/index";
+import { makeStyles } from "@material-ui/core";
+import { ToggleButton } from "@material-ui/lab";
 
-export const useButtonStyles = makeStyles({
-  controlRow: ({ isSelected }) => ({
-    height: 34,
-    padding: 7,
-    display: "flex",
-    justifyContent: "left",
-    background: isSelected && theme.palette.primary.main,
+export const useButtonStyles = makeStyles(theme => ({
+  buttonImage: {
+    width: "1.5rem",
+    height: "1.5rem",
+  },
+  buttonGroupColor: {
+    backgroundColor: theme.palette.common.white,
     "&:hover": {
-      background: isSelected
-        ? theme.palette.primary.main
-        : theme.palette.background.mapControlsHover,
+      backgroundColor: theme.palette.grey["200"],
     },
-  }),
-});
+  },
+  buttonGroupColorSelected: {
+    backgroundColor: theme.palette.primary.light,
+    "&:hover": {
+      backgroundColor: theme.palette.primary.dark,
+    },
+  },
+}));
```

Comment:
In these past few lines, we add the hand icon and we give the draw buttons a style.